### PR TITLE
feat(native): Return the player rank position

### PR DESCRIPTION
**Pull request by Rushaway**
We have to deprecate the old native, it's always return -1 after my tests.

### CODE REVIEW

**Comment by maxime1907** [comment]
just fix the original native no ?

**Comment by Rushaway** [comment]
The original native was suppose to return only if the players are in the top 3 defenders.
This new native return the current position of the player.
The previous native can be done via the new one by just comparing the player position.

File: addons/sourcemod/scripting/include/TopDefenders.inc [hunk 1]
```diff
@@ -14,12 +14,22 @@
...
+#pragma deprecated Use TopDefenderStats() instead.
+native int TopDefenders_IsTopDefender(int client);
+
+/**
+ * Check if the given client is top defender
+ *
+ * @param client     Any valid client index.
+ * @return           Returns -1 if the client is not ingame/ranked, otherwise it returns its top defender position.
+ */
+native int TopDefenderStats(int client);
```
Comment (maxime1907) [inline addons/sourcemod/scripting/include/TopDefenders.inc:32]: TopDefenders_GetClientRank

File: addons/sourcemod/scripting/TopDefenders.sp [hunk 2]
```diff
@@ -1069,4 +1072,22 @@ public int Native_IsTopDefender(Handle plugin, int numParams)
 		}
 	}
 	return -1;
+}
+
+public int Native_TopDefenderStats(Handle plugin, int numParams)
```
Comment (maxime1907) [inline addons/sourcemod/scripting/TopDefenders.sp:1077]: Native_GetClientRank

**Pull request merged**